data/sitesEN.json

```diff
@@ -834,5 +834,16 @@
     "destination_content_path": "/wiki/",
     "destination_platform": "mediawiki",
     "destination_icon": "zeldapedia.png"
+  },
+  {
+    "id": "en-zelda",
```

Comment:
Please update id (can be `en-cyclefrontier`)

Reply:
Ah, and please alphabetize the listing be moving it under the Critical Role wiki. Thank you!

Reply:
Would it be okay to alphabeticalize it under T instead? The full name of the game is The Cycle: Frontier, not Cycle: Frontier

Reply:
Sure, that's fine. Thank you!